src/utils/getBlocksFromTimestamps.ts

```diff
@@ -40,7 +48,7 @@ async function splitQuery(query, localClient, vars, list, skipCount = 100) {
 }
 
 export default async function getBlocksFromTimestamps(
-  timestamps,
+  timestamps: any,
```

Comment:
I think `timestamps` can be `number[]`.

Reply:
@brunobar79 i think only this one was missed